changelog.d/3025.enhance.rst

```diff
@@ -0,0 +1 @@
+Print the newly added quickstart guide after setup.
```

Comment:
This should be `3027.enhance.rst` as it doesn't fix 3025, it can use the standalone number of the PR

Reply:
Addressed, I wasn't really sure what the naming conventions were for changelog commits. Thanks!